ted_sws/core/domain/message_handlers.py

```diff
@@ -20,11 +20,13 @@ def handler_log(log: message.Log):
         msg += ("{title}" + eol).format(title=log.title)
     if log.message:
         if isinstance(log.message, List):
-            msg += (eol + "{messages}" + eol).format(
+            if not log.title:
```

Comment:
I am not sure this module shall be in the Domain.

Reply:
seems technical

Reply:
I don't see other better place for it for the moment. Let's leave it here and will move it afterwards.